src/Commands/StatusController.php

```
@@ -0,0 +1,91 @@
+        $hasher = new Hasher();
+        $result = [];
+
+        foreach ($data['files'] as $destination => $entry) {
+            $absolutePath = $projectRoot . '/' . $destination;
+
+            if (!file_exists($absolutePath)) {
+                $status = 'missing';
+            } else {
+                $currentHash = $hasher->hash($absolutePath);
+                $status = $hasher->equals($currentHash, $entry['hash']) ? 'synced' : 'MODIFIED';
+            }
```

PHPMD (2.15.0)

[error] The method getStatuses uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (undefined)

(ElseExpression)